src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Styles/_Generic.xaml

```diff
@@ -0,0 +1,6 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary Source="ms-appx:///Styles/XamlDisplay.xaml" />
+	</ResourceDictionary.MergedDictionaries>
+</ResourceDictionary>
```

Comment:
generic makes me think about theme/generic.xaml but it doesn't serve this purpose. I think I'd just merge Stles/XamlDisplay.xaml in app.xaml directly, I think we can skip this file.